common/network-common/src/main/java/org/apache/spark/network/buffer/FileSegmentManagedBuffer.java

```diff
@@ -133,7 +132,7 @@ public Object convertToNetty() throws IOException {
     if (conf.lazyFileDescriptor()) {
       return new DefaultFileRegion(file, offset, length);
     } else {
-      FileChannel fileChannel = FileChannel.open(file.toPath(), StandardOpenOption.READ);
+      FileChannel fileChannel = new FileInputStream(file).getChannel();
```

Comment:
@jerryshao I think this is the only line that may reduce the memory pressure for external shuffle service. Right?